types/languages.interfaces.ts

```diff
@@ -93,7 +93,8 @@ export type LanguageKey =
     | 'vala'
     | 'vb'
     | 'wasm'
-    | 'zig';
+    | 'zig'
+    | 'ylc';
```

Comment:
This ought to be in alphabetical order like everything else but we can fix that later :)

Reply:
Oh sorry, does the website update at 00:00 utc?